ArcGISRuntimeSDKQt_QMLSamples/CloudAndPortal/SearchForWebmap/README.md

```diff
@@ -1,23 +1,37 @@
 # Search for web map by keyword
 
-This sample demonstrates how find portal items by using a keyword, and limit the results to items of a certain type - in this case, web maps.
+Find portal items by using a keyword, and limit the results to items of a certain type - in this case, web maps.
 
 ![](screenshot.png)
 
+## Use case
+
+Portals can contain many portal items and at times you may wish to query the portal to find what you're looking for. In this example, we search for webmap portal items using a text search.
+
+## How to use the sample
+
+Enter search terms into the search bar. Once the search is complete, a list is populated with the resultant webmaps. Tap on a webmap to set it to the map view. Scrolling to the bottom of the webmap recycler view will get more results.
+
 ## How it works
+
 1. A `Portal` is created with no credential set, when the app starts, the portal is loaded.
-2. A search bar allows the user to enter a keyword (a tag) to search for - e.g. "usa". 
-3. The search term is passed as the `searchString` for a `PortalQueryParametersForItems` which also has the `itemType` set to `PortalItemType::WebMap`. NOTE if multiple item types are required these can be set via the `types` property. Since webmaps authored prior to July 2nd, 2014 are not supported, a date filter is also applied.
+2. A search bar allows the user to enter a keyword (a tag) to search for * e.g. "usa". 
+3. The search term is passed as the `searchString` for a `PortalQueryParametersForItems` which also has the `itemType` set to `PortalItemType.WebMap`. NOTE if multiple item types are required these can be set via the `types` property. Since webmaps authored prior to July 2nd, 2014 are not supported, a date filter is also applied.
```

Comment:
```suggestion
3. The search term is passed as the `searchString` for a `PortalQueryParametersForItems` which also has the `itemType` set to `Enums.PortalItemTypeWebMap`. NOTE if multiple item types are required these can be set via the `types` property. Since webmaps authored prior to July 2nd, 2014 are not supported, a date filter is also applied.
```